### ISSUE TITLE
Can not hyperclick with React fragments syntax

### ISSUE THREAD

**Issue by elawad** [AsaAyers/js-hyperclick#99]
Not sure if this is related to js-hyperclick.

Using React's `<>foo</>` syntax seems to break hyperclick.

Works here:
<img src="https://user-images.githubusercontent.com/313249/46910100-cbc45900-cf0c-11e8-93ed-2db9ec877a3d.png" width="500px" alt="click working" />

Can not hyperclick using the <> syntax:
<img src="https://user-images.githubusercontent.com/313249/46910099-c6ffa500-cf0c-11e8-8e1b-89ce525f6e81.png" width="500px" alt="click not working" />

**Comment by elawad** [comment]
Sounds good, thanks. Just wanted to report it for clarification.

**Comment by vitexikora** [comment]
How is it looking? Any way we can help?

**Comment by AsaAyers** [comment]
I created a work in progress PR (https://github.com/AsaAyers/js-hyperclick/pull/102) that
1. swaps `babel-core` and `babylon` for `@babel/core`
2. Adds comments where I know things need to change
3. Adds a component using the fragment syntax to one of the test files
  * You can run the tests with `npm test`

Assuming the AST hasn't changed significantly, I think the majority of the work is just figuring out what the new version of `parse` wants. The results seeing seem to conflict with the documentation.

https://github.com/AsaAyers/js-hyperclick/blob/dd5f8b5c189b72f8fd4bda448c2b11d0cdb523c9/lib/core/parse-code.js#L62-L75

I think the best ways to help would be to comment there if you know what's up with the new `parse()`, or make additional PRs that target the `babel-7` branch.

**Comment by vitexikora** [comment]
So I just installed 1.14.0 and it stopped working altogether, be it Babel 6 or 7, using new syntax, or nothing at all, even a standalone simple import. What could be wrong? Console reports nothing, even in dev mode...

**Comment by AsaAyers** [comment]
I just published 1.14.1 to include an option to turn on debugging. Run `localStorage.debug="js-hyperclick*"` in the console to turn it on and `localStorage.debug=""` to turn it off. Here is what it looks like in my console.

![selection_004](https://user-images.githubusercontent.com/324999/48206995-8be26d00-e324-11e8-89da-16f390bb2f2f.png)

**Comment by vitexikora** [comment]
```
js-hyperclick:parse-code parseError +0ms Error: 
Cannot find module '@babel/plugin-syntax-bigint' 
```

It is neccessary to install all possible babel plugins into the project? (I have a lot of them, but certainly not all)
Is it possible for the library to utilize local .babelrc file and have those plugins as a peer dep?

**Comment by AsaAyers** [comment]
Maybe try uninstalling and reinstalling? It doesn't use the versions in your project, it's supposed to install them in `~/.atom/packages/js-hyperclick/node_modules/`

**Comment by vitexikora** [comment]
```
> apm uninstall js-hyperclick
Uninstalling js-hyperclick done

> apm i js-hyperclick@1.14.1
Installing js-hyperclick@1.14.1 to ...\.atom\packages done

> atom .
```
Still the same error message
I have Windows on this machine, if that could be the problem...

**Comment by AsaAyers** [comment]
I just published v1.14.2 that should fix it. [fix](https://github.com/AsaAyers/js-hyperclick/commit/0749736ea400f0d2ad630419c31f057f4fd4e50c)

It actually was looking in your project's `node_modules` instead of `~/.atom/packages/js-hyperclick/node_modules/`. I didn't realize it because I tested `js-hyperclick` by using it to work on its own source code. And that project obviously has all of these dependencies :slightly_smiling_face:. Once I updated `js-hyperclick` on my work computer I was able to reproduce the error. Thank you for your help in debugging this quickly.
